### ISSUE TITLE
Issue with nested pages

### ISSUE THREAD

**Issue by jeremiedb** [LuxDL/DocumenterVitepress.jl#37]
The docs build fails when nested pages are used: 

```julia
pages = [
    "Quick start" => "quick-start.md",
    "Design" => "design.md",
    "Models" => "models.md",
    "API" => "API.md",
    "Tutorials" => [
        "Regression - Boston" => "tutorials/regression-boston.md",
        "Logistic - Titanic" => "tutorials/logistic-titanic.md",
        "Classification - IRIS" => "tutorials/classification-iris.md",
    ]
]
```

results in following error in CI:
```julia
Deploying: ✔
ERROR: LoadError: MethodError: no method matching pagelist2str(::Documenter.Document, ::Pair{String, Any})

Closest candidates are:
  pagelist2str(::Any, ::Pair{String, String})
   @ DocumenterVitepress ~/.julia/packages/DocumenterVitepress/IO55P/src/vitepress_config.jl:75
  pagelist2str(::Any, ::String)
   @ DocumenterVitepress ~/.julia/packages/DocumenterVitepress/IO55P/src/vitepress_config.jl:62
  pagelist2str(::Any, ::Pair{String, <:AbstractVector})
   @ DocumenterVitepress ~/.julia/packages/DocumenterVitepress/IO55P/src/vitepress_config.jl:81
```

However the following works fine: 
```julia
pages = [
    "Quick start" => "quick-start.md",
    "Design" => "design.md",
    "Models" => "models.md",
    "API" => "API.md"
]
```

It looks like this may have been what was being adressed by https://github.com/LuxDL/DocumenterVitepress.jl/pull/35.
However, the above CI example were run with `[4710194d] + DocumenterVitepress v0.0.4`. Maybe the previous fix wasn't part of the latest registered version?

Thanks for the great work, a very promising upgrade to docs format!

**Comment by jeremiedb** [comment]
For reference, here is the reference to the [make.jl](https://github.com/Evovest/NeuroTreeModels.jl/blob/main/docs/make.jl). 
Note that when building locally, it fails with the following: 
```julia
[ Info: RenderDocument: rendering document.
[ Info: DocumenterVitepress: rendering MarkdownVitepress pages.
┌ Warning: DocumenterVitepress: no assets directory found.
└ @ DocumenterVitepress C:\Users\jerem\.julia\packages\DocumenterVitepress\IO55P\src\writer.jl:217


ERROR: LoadError: SystemError: opening file "C:\\Users\\jerem\\OneDrive\\Evovest\\NeuroTreeModels.jl\\docs\\build\\tutorials\\classification-iris.md": No such file or directory
```

I noticed that the `build` folder has some of docs src files directly under `build` while unders are under `build/.documenter`:
![image](https://github.com/LuxDL/DocumenterVitepress.jl/assets/18605903/7e5d1daa-3ca8-495d-8073-b3c4dc8f6b4f)
Not sure it's may relate to the `package.json`: `"docs:dev": "vitepress dev build/.documenter"`? As mentionned, the build is fine on CI (when nested pages are removed), but locally, the `make.jl` fails.

**Comment by asinghvi17** [comment]
Should be fixed in v0.0.6
